test/fixtures/drawing/draw-correct-order-if-undefined-values.js

```diff
@@ -0,0 +1,60 @@
+// https://github.com/chartjs/chartjs-plugin-datalabels/issues/319

```

Comment:
You are testing `null` values, not `undefined` so I would suggest to rename these test files as: 

- `draw-skipped-values-in-correct-order.js`
- `draw-skipped-values-in-correct-order.png`

Reply:
done